specification/m365securityandcompliance/resource-manager/readme.go.md

```diff
@@ -0,0 +1,36 @@
+## Go
+
+These settings apply only when `--go` is specified on the command line.
+
+``` yaml $(go)
+go:
+  license-header: MICROSOFT_APACHE_NO_VERSION
+  clear-output-folder: true
+  namespace: m365securityandcompliance
+```
+
+### Go multi-api
+
+``` yaml $(go) && $(multiapi)
+batch:
+  - tag: package-2021-03-25-preview
+  - tag: package-2021-03-25-preview
+```
+
+### Tag: package-2021-03-25-preview and go
+
+These settings apply only when `--tag=package-2021-03-25-preview --go` is specified on the command line.
+Please also specify `--go-sdk-folder=<path to the root directory of your azure-sdk-for-go clone>`.
+
+``` yaml $(tag)=='package-2021-03-25-preview' && $(go)
+output-folder: $(go-sdk-folder)/services/$(namespace)/mgmt/2021-03-25-preview/$(namespace)
```

Comment:
Preview packages must be generated to the preview subdirectory, as said in the CI error message
```suggestion
output-folder: $(go-sdk-folder)/services/preview/$(namespace)/mgmt/2021-03-25-preview/$(namespace)
```